docs/src/wallets/test-wallets.md

```diff
@@ -27,6 +27,17 @@ You can create a test wallet containing multiple assets (including the base asse
 - coins: `Vec<(UtxoId, Coin)>` has num_assets * coins_per_assets coins (UTXOs)
 - asset_ids: `Vec<AssetId>` contains the num_assets randomly generated `AssetId`s (always includes the base asset)
 
+## Setting up a test wallet with multiple custom assets
```

Comment:
I think you should also rename the previous section with `with multiple random assets` or something.